### PR TITLE
Huybrand

### PR DESCRIPTION
**Pull request by NguyenHuy1812**
I add in a tralier feature for your movie Phil, hope this work for you

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,7 +4,10 @@ import './App.css';
 import moment from 'moment';
 import { css } from '@emotion/core';
 import { BeatLoader } from 'react-spinners';
-import InputRange from 'react-input-range'
+import InputRange from 'react-input-range';
+import Modal from 'react-modal';
+import YouTube from '@u-wave/react-youtube';
+
```
Comment (Nguyephi) [inline src/App.js]: Extra space!

File: src/App.js [hunk 2]
```diff
@@ -28,8 +31,12 @@ class App extends React.Component {
       isSearched: false,
       // hasSortingType: false,
       isLoading: true,
+      showModal: false,
+      id: null
 
     }
+
+
```
Comment (Nguyephi) [inline src/App.js]: ?!?!?!?!

File: src/App.js [hunk 3]
```diff
@@ -47,6 +54,7 @@ class App extends React.Component {
       pageNumber: pageNumber + 1,
       movies: filterSearchResults,
       isLoading: false,
+
```
Comment (Nguyephi) [inline src/App.js]: more empty spaces??

Comment (NguyenHuy1812) [reply src/App.js]: Already fix that

**Pull request merged**